packages/manager/modules/pci/src/projects/project/activate/activate.html

```diff
@@ -5,18 +5,32 @@
             data-ng-if="$ctrl.isTrustedZone"
         ></ovh-manager-trusted-nic-label>
     </div>
-    <div class="d-flex flex-md-row-reverse flex-sm-column">
+    <div class="d-flex flex-md-row-reverse flex-column">
         <div class="p-4 oui-background-p-050">
             <div>
-                <h1
-                    data-translate="pci_projects_project_pci_banner_promotion_title"
-                ></h1>
-                <p
-                    data-translate="pci_projects_project_pci_banner_promotion_description_1"
-                ></p>
-                <p
-                    data-translate="pci_projects_project_pci_banner_promotion_description_2"
-                ></p>
+                <div data-ng-if="!$ctrl.discoveryPromotionVoucherAmount">
+                    <h2
+                        class="font-weight-bolder"
```

Comment:
**question:** Do we really need this?

Reply:
@anooparveti 
Yes.
After multiple back-&-forth with the UI designer, the decision to have this title displayed with a bold font weight was maintained.
The typography of an H1 title have a light font weight, which require this bootstrap class.